docs/cli.md

```diff
@@ -78,188 +369,320 @@ by applying key-value pairs that will be matched with the resource labels:
 $ oc get pods -s template=production
 ```
 
-This command will return only pods whose `labels` include `"template": "production"`
+This command will return only pods whose `labels` include `"template": "production"`.
 
-oc create
---------------
+See also [`oc describe`](#oc-describe).
 
-This command can be used to create resources. It does not require pointers about
-what resource it should create because it reads it from the provided JSON/YAML.
-After successful creation, the resource name will be printed to the console.
+### oc describe
 
-#### Examples
+This functions similar to [`oc get`](#oc-get), but also includes other information related to the specified resource.
 
 ```
-$ oc create -f pod.json
-$ cat pod.json | oc create -f -
-$ oc create -f http://server/pod.json
+$ oc describe service frontend
+```
+
+### oc edit
+
+This starts an editor opened to the YAML representation of the specified object.
+When the editor exits, the object is updated.
+You can specify the editor through environment variables `OSC_EDITOR`, `GIT_EDITOR`, or `EDITOR`.
```

Comment:
@soltysh Thanks for catching this.  Fixed in: https://github.com/tnguyen-rh/origin/commit/69679ce4ded53f5b690ff033043c04818b09a4ad